package.json

```diff
@@ -24,7 +24,7 @@
     "module_name": "apm_bindings",
     "module_path": "./dist/{node_napi_label}",
     "staging_host": "https://rc-files-t2.s3-us-west-2.amazonaws.com",
-    "production_host": "https://apm-bruce-production.s3.amazonaws.com",
+    "production_host": "https://files.appoptics.com.s3.amazonaws.com",
```

Comment:
Hmm, I get an SSL error about certificate subject name with `curl -i https://files.appoptics.com.s3.amazonaws.com`.  Need to check if this is actually the correct S3 bucket endpoint...

Reply:
Maybe because the production S3 is configured with a hostname to be `https://files.appoptics.com` instead.  I wonder if there's a quick way for you to check (will ping with credentials).

Reply:
the problem was that the wildcard in the cert only matches one level deep on the domain name. because `files.appoptics.com` uses dots it didn't match `*.s3.amazonaws.com`. if the cert had added the altname of `files.appoptics.com.s3.amazonaws.com` (not sure that's legal) it would have worked. but dot separates levels of the host name and wildcards are only valid for one level.

but the simplest fix was to make a vanilla bucket, `appoptics-binaries.s3.amazonaws.com` and use that.

i have published to it and installed from it.